recipes/perl-text-fuzzy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.29" %}
+{% set sha256 = "3df5cfd2ca1a4c5ca7ff7bab3cc8d53ad2064e134cbf11004f3cf8c4b9055bff" %}
+
+# regex to use for Windows build hack
+#{% set win_patch = "s|C:\\\\strawberry\\\\c|$ENV{LIBRARY_PREFIX}\\\\mingw-w64|g" %}
```

Comment:
You can remove this since you removed the line where the variable is used.

Reply:
Got it